### DISCUSSION TITLE
Edu Mail Rejection

### DISCUSSION THREAD

**Discussion by 0x5900FX** [community/community#129782] [GitHub Education]
### Hi everyone,

My college doesn't provide edu mail. How am i supposed to verify it. When i try to verify without selecting college and submit my college id card and docs. I get rejected. Help me with this

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: Orchestrator
- **Step**: S04
- **Link**: https://github.com/CrossTerra/monorepo/pull/2891#issuecomment-4579560918
- **Summary**: Reviewer returned INCOMPLETE (brittle line-number reference in S02 prose); added delta step S04 to PR #2891 and invoked developer.

**Comment by MahdiJDS** [comment]
You're not necessarily required to have an `.edu` email address to be approved for GitHub Education.

Many students are verified using alternative proof of enrollment when their institution does not provide academic email addresses.

A few things that may help:

* Make sure the school you attend is listed correctly in the application.
* Submit a clear, high-resolution image of your student ID.
* Include additional documentation if available, such as:

  * enrollment certificate
  * current class schedule
  * tuition receipt
  * official student portal screenshot
  * other documents showing your name, institution, and current enrollment status

One common reason for rejection is that the documentation does not clearly show that you are currently enrolled, or the information is difficult to read.

If your application was rejected, check whether the rejection email provides any specific reason. That can help determine what additional documentation is needed.

Also, what country are you studying in, and what type of documents did you submit with your application? Someone may be able to suggest which forms of proof have been accepted from students at similar institutions.
